### PR TITLE
Added derive(ToCss) for Content and removed manual implementation

### LINKED ISSUE #19967: Derive ToCss for Content.
After #19957 lands, there's going to be an easy way to remove some lines of code:

```rust
impl ToCss for Content {
    fn to_css<W>(&self, dest: &mut CssWriter<W>) -> fmt::Result
        where W: Write,
    {
        match *self {
            Content::Normal => dest.write_str("normal"),
            Content::None => dest.write_str("none"),
            #[cfg(feature = "gecko")]
            Content::MozAltContent => dest.write_str("-moz-alt-content"),
            Content::Items(ref content) => {
                let mut iter = content.iter();
                iter.next().unwrap().to_css(dest)?;
                for c in iter {
                    dest.write_str(" ")?;
                    c.to_css(dest)?;
                }
                Ok(())
            }
        }
    }
}
```

This can be derived on the `Content` definition using `#[css(iterable)]` introduced in 09a3db8657961a0a4e20b7f296c1339175ab4c76 (#19583).

### PR DESCRIPTION
**Pull request by nupurbaghel**
---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [ ] These changes fix #19967 (https://github.com/servo/servo/issues/19967). 

- [x] These changes do not require tests because they perform code cleanup

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/20019)

### CODE REVIEW

File: components/style/values/specified/counters.rs [hunk 1]
```diff
@@ -82,7 +82,7 @@ type CounterStyleType = CounterStyleOrNone;
 /// The specified value for the `content` property.
 ///
 /// https://drafts.csswg.org/css-content/#propdef-content
-#[derive(Clone, Debug, Eq, MallocSizeOf, PartialEq, ToComputedValue)]
+#[derive(Clone, Debug, Eq, MallocSizeOf, PartialEq, ToComputedValue, ToCss)]
 pub enum Content {
```
Comment (emilio) [inline components/style/values/specified/counters.rs]: I think the only thing missing here is `#[css(iterable)]` right before the `Items` variant.

Comment (nupurbaghel) [reply components/style/values/specified/counters.rs]: oh, alright. thanks :)

**Comment by emilio** [comment]
@bors-servo r+

Looks great, thanks so much! :)

**Pull request merged**